src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -103,18 +103,29 @@ public SortClause deserialize(JsonParser parser, DeserializationContext ctxt)
         sortExpressions.add(exp);
         break;
       } else {
+        if (path.isBlank()) {
+          throw new JsonApiException(
+              ErrorCode.INVALID_SORT_CLAUSE_PATH,
+              String.format(
+                  "%s: sort clause path must be represented as not-blank string",
```

Comment:
Minor suggestion: I think "sort clause path cannot be a blank string" is easier to read